roles/network_plugin/calico/rr/tasks/main.yml

```diff
@@ -22,5 +22,5 @@
 
 - name: Calico-rr | Set label for route reflector
   command: >-
-    {{ bin_dir }}/calicoctl.sh label node {{ inventory_hostname }}
+    {{ bin_dir }}/calicoctl.sh label node {{ kube_override_hostname }}
```

Comment:
Won't things fail if it expects a node, and you provide an empty string?

Reply:
@holmsten @mattymo I've tested and it indeed generate an error if you don't provide a node/hostname